packages/ui-kit/src/components/code-block.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
+import { css } from '@emotion/core';
 import Tooltip from '@commercetools-uikit/tooltip';
 import SpacingsInline from '@commercetools-uikit/spacings-inline';
 import { ClipboardIcon } from '@commercetools-uikit/icons';
+import theme from 'prism-react-renderer/themes/nightOwl';
```

Comment:
FYI: previously we were using the "dracula" theme but I find this to be a bit "nicer".

Reply:
https://github.com/sdras/night-owl-vscode-theme
vs. 
https://github.com/dracula/visual-studio-code

I'm not a fan of mixing design changes with refactors but agree that nightOwl matches our CI better than dracula (I always found that strong violet a mismatch)